src/lib/utilities.js

```diff
@@ -278,3 +278,6 @@ export function filterByEnvironment(development, beta, production) {
   }
   return development;
 }
+
+export const nothingAllowedRobotsTxt = 'User-agent: *\nDisallow: / ';
```

Comment:
Just to conform with the format Google and Facebook etc. follow just replace the space at the end here with a newline character, so it becomes `Disallow: /\n`